geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/FeatureExporter.scala

```diff
@@ -171,19 +155,18 @@ class DelimitedExport(writer: Writer, format: Formats, attributes: Option[String
     logger.info(s"Exported $count features")
   }
 
-  def stringify(o: Object): String = o match {
-    case null    => ""
-    case d: Date => dateFormat.format(d)
-    case _       => o.toString
+  def stringify(o: Object): Object = o match {
+    case g: Geometry => WKTUtils.write(g)
+    case d: Date     => GeoToolsDateFormat.print(d.getTime)
+    case _           => o
```

Comment:
Not specifically this PR but I guess we should probably deal with List and Map at some point safely here. Haven't looked to see what they do recently but I assume that java.util.Map might not be friendly :)